extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -504,8 +504,7 @@ export class ConsentUI {
     if (!elementByTag(this.parent_, 'placeholder')) {
       insertAfterOrAtStart(
```

Comment:
Nit: it'd probably be more clear to have an `insertFirst` helper.

Reply:
Fixed